gui/elements/qt/_controls.py

```diff
@@ -0,0 +1,10 @@
+from PyQt5.QtCore import Qt
+from PyQt5.QtWidgets import QHBoxLayout, QSlider
+
+
+class QtControls(QHBoxLayout):
+    def __init__(self):
+        super().__init__()
+
+        self.addWidget(QSlider(Qt.Vertical))
```

Comment:
Just a note to passerby that this will be made customizable as well (possibly in a future pull request).

Reply:
Yes, my plan now was just to leave this here as a non-functional placeholder and then future PRs can address it. If that is undesirable I could comment it out or delete it.